.vsts-dotnet-ci.yml

```diff
@@ -65,7 +65,7 @@ jobs:
     name: Windows_x64_NetCoreApp21
     buildScript: build.cmd
     pool:
-      name: Hosted VS2017
+      name: NetCorePublic-Pool
```

Comment:
Do we need to say

```
queue: buildpool.windows.10.amd64.vs2017.open
```

like we have above?

Reply:
what about for x86?

---
In reply to: [338675654](https://github.com/dotnet/machinelearning/pull/4375#discussion_r338675654) [](ancestors = 338675654)

Reply:
We run our `x86` build on a 64-bit machine/OS. We just use a 32-bit `dotnet.exe` to do the build and run the tests.

Reply:
thanks. I have the made the change.